### PR TITLE
Improve error

### PR DESCRIPTION
**Pull request by stefanpenner**

### CODE REVIEW

**Comment by Turbo87** [comment]
asdf?

**Comment by rwjblue** [comment]
Looks like CI failures are due to incompatibilities in how `(function() {}).toString()` works on various node versions.

```
       {
         "errors": [
           "name: unknown, located at: unknown"
      -    "name: unknown, located at: unknown, src: function () {}"
      +    "name: unknown, located at: unknown, src: function() {}"
         ]
         "isParallelizable": false
       }
```

**Comment by stefanpenner** [comment]
I'm currently debugging some of these issues locally, and updating this PR as I find ways to make it more actionable

**Comment by stefanpenner** [comment]
Not perfect, but pretty actionable so far: 

screen shot 2018-07-19 at 2 39 06 pm

File: index.js [hunk 1]
```diff
@@ -71,10 +71,24 @@ function Babel(inputTree, _options) {
...
 
-  if ((this.throwUnlessParallelizable || process.env.THROW_UNLESS_PARALLELIZABLE) && transformIsParallelizable(options) === false) {
-    throw new Error(this.toString() + ' was configured to `throwUnlessParallelizable` and was unable to parallelize an plugin. Please see: https://github.com/babel/broccoli-babel-transpiler#parallel-transpilation for more details');
+  let result = transformIsParallelizable(options);
+  let isParallelizable = result.isParallelizable;
+  let errors = result.errors;
+
+  if ((this.throwUnlessParallelizable || process.env.THROW_UNLESS_PARALLELIZABLE) && isParallelizable === false) {
+    throw new Error(this.toString() +
+      ' was configured to `throwUnlessParallelizable` and was unable to parallelize an plugin. \nplugins:\n' + joinCount(errors) + '\nPlease see: https://github.com/babel/broccoli-babel-transpiler#parallel-transpilation for more details');
```
Comment (mikrostew) [inline index.js]: Nit: "parallelize a plugin"

**Comment by stefanpenner** [comment]
- [ ] fix: a plugin of the form: `[serializable, nonSerializable]` will incorrectly display `serializable` as the problem, not `nonSerializable`

**Pull request merged**